src/Behat/Behat/Hook/Call/RuntimeFeatureHook.php

```diff
@@ -30,7 +32,7 @@ abstract class RuntimeFeatureHook extends RuntimeFilterableHook
      *
      * @param string      $scopeName
      * @param null|string $filterString
-     * @param callable    $callable
+     * @param callable|array $callable
```

Comment:
I suggest using `array{class-string, string}` to restrict the shape of the accepted array for the case of context instance methods.